web/src/cypress/support/select-by-label.ts

```diff
@@ -22,12 +22,9 @@ type findByLabelFn = (label: string) => Cypress.Chainable
 type multiRemoveByLabelFn = (label: string) => Cypress.Chainable
 
 function selectByLabel(sub: any, label: string): Cypress.Chainable {
-  return findByLabel(sub, label)
-    .click()
-    .get('[data-cy=select-dropdown]')
-    .should('not.exist')
-    .get('ul[role=listbox]')
-    .should('not.exist')
+  findByLabel(sub, label).click()
+
+  return cy.get('[data-cy=select-dropdown]').should('not.exist')
```

Comment:
These look equivalent, other than we no longer wait for the listbox to disappear, why the change?